homeassistant/components/homekit/type_media_players.py

```diff
@@ -249,14 +262,20 @@ def __init__(self, *args):
 
         self.sources = []
 
+        self._event_scripts = {}
```

Comment:
We shouldn't take in scripts as config, we should fire events and so allowing anyone to do anything they want, like capture the event in an automation.